content/nginx/deployment-guides/amazon-web-services/high-availability-network-load-balancer.md

```diff
@@ -45,192 +43,214 @@ NGINX Plus provides both reverse‑proxy features and load‑balancing features
-This Appendix provides links to instructions for creating EC2 instances with the names used in this guide, and then installing and configuring NGINX Open Source and NGINX Plus on them:
+Use links in this appendix to create EC2 instances with names used in the above instructions. Then, use steps below to install and configure NGINX Open Source and NGINX Plus on those instances.
+
+You can either complete these individual steps:
 
-- [Creating EC2 Instances and Installing the NGINX Software](#create-instance-install-nginx)
-- [Configuring NGINX Open Source on the Web Servers](#configure-web-servers)
-- [Configuring NGINX Plus on the Load Balancers](#configure-load-balancers)
+- [Create EC2 Instances and Install the NGINX Software](#create-instance-install-nginx)
+- [Configure NGINX Open Source on the Web Servers](#configure-web-servers)
+- [Configure NGINX Plus on the Load Balancers](#configure-load-balancers)
 
-As an alternative to creating and configuring instances individually, you can use our Packer and Terraform scripts to completely automate the process:
+OR
 
-- [Automating Instance Setup with Packer and Terraform](#create-instances-automated)
+Use our Packer and Terraform scripts to completely automate the process:
 
-After completing the instructions, you have completed the prerequisites for this guide and can continue to [Configuring an AWS Network Load Balancer](#nlb-configure).
+- [Automate Instance Setup with Packer and Terraform](#create-instances-automated)
+
+Once you have created and configured the EC2 instances, your prerequisites are complete. Continue to [Configure an AWS Network Load Balancer](#nlb-configure).
 
 <span id="create-instance-install-nginx"></span>
-### Creating EC2 Instances and Installing the NGINX Software
+#### Create EC2 Instances and Install the NGINX Software
 
-The deployment in this guide uses six EC2 instances: two instances running NGINX Plus that load balance traffic to four instances running NGINX Open Source as a web server. The four NGINX Open Source instances are deployed in two pairs, each pair running a different app.
+The deployed solution in these instructions uses six EC2 instances. Two instances run NGINX Plus. These load balance traffic to the other four instances, which run NGINX Open Source as a web server. The four NGINX Open Source instances deploy in two pairs; each pair runs a different app.
 
-<span style="white-space: nowrap;">Step‑by‑step</span> instructions for creating EC2 instances and installing NGINX Open Source and NGINX Plus are provided in our deployment guide, [Creating Amazon EC2 Instances for NGINX Open Source and NGINX Plus]({{< relref "ec2-instances-for-nginx.md" >}}).
+<span style="white-space: nowrap;">Step‑by‑step</span> instructions for creating EC2 instances and installing NGINX Open Source and NGINX Plus are available. Refer to our deployment guide, [Creating Amazon EC2 Instances for NGINX Open Source and NGINX Plus]({{< relref "ec2-instances-for-nginx.md" >}}).
```

Comment:
```suggestion
*Step‑by‑step* instructions for creating EC2 instances and installing NGINX Open Source and NGINX Plus are available. Refer to our deployment guide, [Creating Amazon EC2 Instances for NGINX Open Source and NGINX Plus]({{< ref "/nginx/deployment-guides/amazon-web-services/ec2-instances-for-nginx.md" >}}).
```

We use absolute paths for everything anyway: I'm slowly swapping `relref` to `ref` everywhere to reflect this fact.